cypress/integration/IconLogo.spec.py.js

```diff
@@ -0,0 +1,26 @@
+describe('Icon Logo', () => {
+  Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here prevents Cypress from
+    // failing the tests due to uncaught errors
+    return false;
+  });
+
+  it('Displays the organization logo when the user belongs to an organization', () => {
+    cy.visit('http://localhost:3001/login');
+    cy.get('#userName').type('test1');
+    cy.get('#password').type('EoCAyCPpW0');
+    cy.contains(/log/i).click();
+    cy.contains(/earnings/i).click();
+    cy.wait(100);
+    cy.get('img').should('have.attr', 'alt', 'organization logo');
+  });
+  it('Displays the Greenstand logo when the user does not belongs to an organization', () => {
+    cy.visit('http://localhost:3001/login');
+    cy.get('#userName').type('admin');
+    cy.get('#password').type('8pzPdcZAG6&Q');
```

Comment:
We need to find another way to run this test. We can't commit credentials to the repository like this.
Can we somehow prompt for the credentials when running the test or take them from a local config file that's not committed?

Reply:
I do apologize! I came across an older committed integration test that contained credentials and thought this was a common practice, I should have known better. 

I removed all credentials I found on Cypress tests and created a `cypress.env.json` file to store credentials to use on tests, this file has also been added `.gitignore`.

Reply:
No worries!